test/support/pages/SettingsPage.js

```diff
@@ -45,35 +46,18 @@ define(function (require) {
 
     selectTimeFieldOption: function (selection) {
       var self = this;
-      return common.tryForTime(defaultTimeout * 2, function () {
-        return self.getTimeFieldNameField().click();
-      })
+
+      return self.getTimeFieldNameField().click()
       .then(function () {
-        return self.getTimeFieldNameField().click();
+        return self.getTimeFieldNameField().click()
```

Comment:
I was wondering the same thing, so I removed it at one point, but it doesn't work unless the click is called twice.

I'm not entirely sure why though. Whoever wrote that originally probably came across the same issue and use this as a solution since it does reliably work.